moment/moment-node.d.ts

```diff
@@ -465,7 +465,7 @@ declare module moment {
         max(moments: Moment[]): Moment;
 
         normalizeUnits(unit: string): string;
-        relativeTimeThreshold(threshold: string, limit: number): void;
+        relativeTimeThreshold(threshold: string, limit?: number): void|number;
```

Comment:
It might be better to split this out into two functions since the actual usage and return values don't fully match what this defines.

```js
moment.relativeTimeThreshold('m') == 45;
moment.relativeTimeThreshold('q') == false;

moment.relativeTimeThreshold('m', 46) == true;
moment.relativeTimeThreshold('q', 46) == false;
```